infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -182,6 +183,19 @@ public void handleTrimMarkRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServ
         r.sendResponse(ctx, msg, CorfuMsgType.TRIM_MARK_RESPONSE.payloadMsg(streamLog.getTrimMark()));
     }
 
+    @ServerHandler(type = CorfuMsgType.QUOTA_USED_REQUEST)
+    public void handleQuotaUsedRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
+        log.debug("handleLogFileSizeRequest: received a request {}", msg);
+        r.sendResponse(ctx, msg, CorfuMsgType.QUOTA_USED_RESPONSE.payloadMsg(streamLog.getLogSizeQuota().getUsed()));
+    }
+
+    @ServerHandler(type = CorfuMsgType.SEGMENT_SIZE_REQUEST)
+    public void handleSegmentSizeRequest(CorfuPayloadMsg<SegmentSizeRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
+        log.debug("handleSegmentSizeRequest: received a request {}", msg);
```

Comment:
Can you collapse these two RPCs into just one. I think its useful to have a generic LogStats object that can be queried.

Reply:
It can be done another PR when we need more stats.